packages/fxa-content-server/tests/functional/recovery_key.js

```diff
@@ -81,15 +81,67 @@ registerSuite('Recovery key', {
 
   tests: {
     'can revoke recovery key': function() {
-      return this.remote
-        .then(click(selectors.RECOVERY_KEY.CONFIRM_REVOKE))
-        .then(
-          testElementExists(selectors.RECOVERY_KEY.CONFIRM_REVOKE_DESCRIPTION)
-        )
+      const remote = this.remote;
+      let secondKey;
+      return (
+        this.remote
+          .then(click(selectors.RECOVERY_KEY.CONFIRM_REVOKE))
+          .then(
+            testElementExists(selectors.RECOVERY_KEY.CONFIRM_REVOKE_DESCRIPTION)
+          )
+          .then(click(selectors.RECOVERY_KEY.CONFIRM_REVOKE_OK))
+          .then(testElementExists(selectors.RECOVERY_KEY.STATUS_DISABLED))
 
-        .then(click(selectors.RECOVERY_KEY.CONFIRM_REVOKE_OK))
-        .then(testElementExists(selectors.RECOVERY_KEY.STATUS_DISABLED))
-        .end();
+          //create a new recovery key
```

Comment:
Some comments have spaces before the text and some dont. I think it looks better with a space before the text.

Reply:
made the changes. Thank you! @vbudhram